cmd/utils/flags.go

```diff
@@ -1137,7 +1141,8 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 				Fatalf("Failed to create developer account: %v", err)
 			}
 		}
-		if err := ks.Unlock(developer, ""); err != nil {
+		password := ctx.GlobalString(DeveloperPasswordFlag.Name)
```

Comment:
You'd need to use this flag a bit higher too, when creating the developer account to remain consistent on all code paths.

Reply:
@karalabe Thanks, updated my pull request